src/transactions/transaction_db.rs

```diff
@@ -970,6 +997,16 @@ impl TransactionDB<MultiThreaded> {
             .cloned()
             .map(UnboundColumnFamily::bound_column_family)
     }
+
+    /// Drops the column family with the given name by internally locking the inner column
+    /// family map. This avoids needing `&mut self` reference
+    pub fn drop_cf(&self, name: &str) -> Result<(), Error> {
+        if let Some(cf) = self.cfs.cfs.write().unwrap().remove(name) {
+            self.drop_column_family(cf.inner, cf)
```

Comment:
good point! I think this will (correctly) return an error if the user drops a column family from one thread while continuing to use it from another. Overall seems safe.